.gitignore

```diff
@@ -1,2 +1,2 @@
 /node_modules
-/lib/globals-set.js
+# /lib/globals-set.js
```

Comment:
Let's add a comment (i.e. line prefixed with `#`) explaining why this line is commented out for now.